openhub-ext/src/main/resources/openhub.properties

```diff
@@ -0,0 +1,40 @@
+# ===================================================================
+# OpenHub Project specific configuration
+# ===================================================================
+
+# ===============================
+# = CORE PROPERTIES
+# ===============================
+info.app.name=@project.name@
+info.app.description=@project.description@
+info.app.version=@project.version@ [${info.app.build.git.commit.id}]
+info.app.core.version=@openhub.version@
+info.app.build.number=@git.build.version@
+info.app.build.timestamp=@git.build.time@
+info.app.build.git.branch=@git.branch@
+info.app.build.git.commit.id=@git.commit.id.abbrev@
+info.app.build.git.commit.time=@git.commit.time@
+
+## DATASOURCE (DataSourceAutoConfiguration & DataSourceProperties)
+# name of the data source
+spring.datasource.name=OHF-RI
+# populate using data.sql
+spring.datasource.initialize=false
+# a charset for reading SQL scripts
+spring.datasource.sql-script-encoding=UTF-8
+spring.datasource.driver-class-name=org.h2.Driver
+# see http://www.h2database.com/html/cheatSheet.html for possibilities
+# we activate mixed-mode to support multiple processes can access the same database
+spring.datasource.url=jdbc:h2:~/openHubDB_ri;DB_CLOSE_ON_EXIT=FALSE;AUTO_SERVER=TRUE
+spring.datasource.username=sa
+spring.datasource.password=
+spring.datasource.hikari.pool-name=OHFRI-HikariCP
+spring.datasource.hikari.maximum-pool-size=10
+spring.datasource.validation-query=select 'connection test'
+
+# ===============================
+# = EXTERNAL SYSTEM CONFIGURATION
+# ===============================
+
+# exchange service url
+out.exchange.rate.url = api.fixer.io/latest
```

Comment:
This should be added in commit where it is used - implementation of Exchange route